tests/pipeline/test_pipeline.py

```diff
@@ -235,6 +235,51 @@ def test_with_custom_config(self, snapshot: SnapshotTest):
 
         snapshot.assert_match(enriched_pipeline, "test-pipeline")
 
+    def test_with_custom_config_with_absolute_defaults_path(
+        self, snapshot: SnapshotTest
+    ):
+        with open(RESOURCE_PATH / "custom-config/config.yaml", "r") as rel_config_yaml:
+            config_dict = yaml.safe_load(rel_config_yaml)
+        defaults_path = (
+            RESOURCE_PATH / "custom-config" / Path(config_dict["defaults_path"])
+        )
```

Comment:
let's just use the folder name directly
```suggestion
        defaults_path = (RESOURCE_PATH / "no-topics-defaults").absolute()
```